# goal definition
geometry_msgs/PoseStamped targetPose
---
# result definition
geometry_msgs/PoseStamped endPose
---
# feedback definition
geometry_msgs/PoseStamped currentPose
